core/src/main/java/com/linecorp/armeria/common/ResponseHeadersBuilder.java

```diff
@@ -42,6 +42,15 @@
      */
     ResponseHeadersBuilder status(HttpStatus status);
 
+    /**
+     * Sets the {@code "set-cookie"} header.
+     * @param cookies the specified cookies.
+     * @return {@code this}
+     */
+    default ResponseHeadersBuilder setCookie(Cookie... cookies) {
```

Comment:
How about adding another version that takes `Iterable<Cookie>`?

Reply:
Thank you for comment. I add another version that takes `Iterable<? extends Cookie>`. Can you review it once again? 🙏